lib/posix-sysinfo/README.md

```diff
@@ -0,0 +1,65 @@
+# `posix-sysinfo`: Unikraft's System Information Internal Library
+
+`posix-sysinfo` is an internal library of Unikraft that provides a similar interface to the Linux system information related syscalls (`sysinfo`, `uname`, etc.).
+
+The `struct sysinfo` and `struct utsname` structures follow the Linux conventions.
```

Comment:
Could you provide to where these structured are defined? I think it would be useful for a quick check of the fields.